package.json

```diff
@@ -25,17 +34,18 @@
     "standalone:replay": "REPLAY=replay cozy-konnector-standalone src/index.js",
     "build": "webpack",
     "format": "prettier --write '**/*.{js,json}'",
-    "precommit": "pretty-quick --staged",
-    "deploy":
-      "git-directory-deploy --directory build/ --branch build --repo=https://github.com/konnectors/cozy-konnector-mgen.git",
-    "deploy:prod":
-      "git-directory-deploy --directory build/ --branch latest --repo=https://github.com/konnectors/cozy-konnector-mgen.git"
+    "precommit": "eslint . && pretty-quick --staged",
```

Comment:
Why is the precommit here different from the one in ameli ?

https://github.com/konnectors/cozy-konnector-maif/pull/33/files

Reply:
parce que j'ai pas encore mis le linter sur ameli ;)